src/cobra/flux_analysis/variability.py

```diff
@@ -26,6 +26,29 @@
 configuration = Configuration()
 
 
+class _FakeProgress:
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *args):
+        pass
+
+    def add_task(self, *args, **kwargs):
+        return object()
```

Comment:
In rich, `add_task` returns an integer ID. So maybe just `return 0` here.

Reply:
Do you need this at all? In rich Progress.add_task already has a visible flag that hides the progress bar. Does that have too much overhead?

Reply:
Didn't realize there was that option. Sounds good 🙂